CHANGELOG.md

```diff
@@ -11,11 +11,16 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Added `State_Diag%Archive_SatDiagnEdgeCount` field
 - Added `State_Diag%Archive_SatDiagnEdge` field
 - Added routine `SatDiagn_or_SatDiagnEdge` in `History/history_utils_mod.F90`
+- Added KPP standalone interface (archives model state to selected locations)
+- Added `https://github/geoschem/KPP-Standalone` as a Git submodule (`geoschem-dev` branch)
```

Comment:
@lizziel: The main branch of the [KPP-Standalone repo](https://github.com/geoschem/KPP-Standalone) (which is called `kpp-standalone`) contains the `gckpp_*F90` files that were built by KPP.  For the interface to GEOS-Chem, I didn't want to carry these, so I created a new branch `geoschem-dev` (akin to to the `gchp-dev` branch in the MAPL fork) so that I wouldn't touch the code in the `kpp-standalone` branch.  

I've merged the `kpp-standalone` branch into the `geoschem-dev` branch (but removed the gckpp* etc files).  That way the `kpp_standalone.F90` and `kpp_standalone_init.F90` codes are up to date.

Do you have any other recommendations how we can manage this?  I was thinking something like how you manage the MAPL repo but am open to other ideas.